spacy/tests/test_cli_app.py

```diff
@@ -37,6 +38,8 @@ def test_benchmark_accuracy_alias():
     # Verify that the `evaluate` alias works correctly.
     result_benchmark = CliRunner().invoke(app, ["benchmark", "accuracy", "--help"])
     result_evaluate = CliRunner().invoke(app, ["evaluate", "--help"])
-    assert result_benchmark.stdout == result_evaluate.stdout.replace(
-        "spacy evaluate", "spacy benchmark accuracy"
+    assert re.sub(r"\s+", " ", result_benchmark.stdout) == re.sub(
+        r"\s+",
+        " ",
+        result_evaluate.stdout.replace("spacy evaluate", "spacy benchmark accuracy"),
```

Comment:
Could we have this as a utility function in `tests\util.py` ?

Reply:
I don't see a whole lot of advantage in moving that to a util method?

Reply:
it's just that the `re.sub` command is copy/pasted in the current implementation, making it brittle to future changes IMO

Reply:
I don't really care...